screens/Whirlpools/components/Whirlpool.tsx

```diff
@@ -25,6 +25,7 @@ const Whirlpool = ({
 
   useEffect(() => {
     const fetchPoolData = async () => {
+      // const poolData = { tickCurrentIndex: 123 }
       const poolData = (await orcaClient.getPool(whirlpool.address)).getData();
```

Comment:
to get error messages (as opposed to app crashing silently): 
1. comment out orcaClient (l.29) and instead use hardcoded `poolData` 
2. build app normally (`yarn start`) 
3. comment out l.28 and use orcaClient again (without reloading app manually)